beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -640,6 +641,132 @@ func (bs *Server) validateEquivocation(blk interfaces.ReadOnlyBeaconBlock) error
 	return nil
 }
 
+// ListBlockHeaders retrieves block headers matching given query. By default it will fetch current head slot blocks.
+func (bs *Server) ListBlockHeaders(w http.ResponseWriter, r *http.Request) {
```

Comment:
Let's follow the convention of naming the receiver `s` in HTTP handlers and rename the function to `GetBlockHeaders`.